pennylane_lightning/lightning_kokkos/lightning_kokkos.py

```diff
@@ -71,6 +76,173 @@ def _kokkos_configuration():
     return print_configuration()
 
 
+def simulate(  # pylint: disable=unused-argument
+    circuit: QuantumScript,
+    state: LightningStateVector,
+    mcmc: dict = None,
+    postselect_mode: str = None,
+) -> Result:
+    """Simulate a single quantum script.
+
+    Args:
+        circuit (QuantumTape): The single circuit to simulate
+        state (LightningStateVector): handle to Lightning state vector
+        mcmc (dict): Dictionary containing the Markov Chain Monte Carlo
+            parameters: mcmc, kernel_name, num_burnin. Descriptions of
+            these fields are found in :class:`~.LightningQubit`.
+        postselect_mode (str): Configuration for handling shots with mid-circuit measurement
+            postselection. Use ``"hw-like"`` to discard invalid shots and ``"fill-shots"`` to
+            keep the same number of shots. Default is ``None``.
+
+    Returns:
+        Tuple[TensorLike]: The results of the simulation
+
+    Note that this function can return measurements for non-commuting observables simultaneously.
+    """
+    return 0
+    # if mcmc is None:
```

Comment:
Is it still WIP?

Reply:
Hi @multiphaseCFD. Yes, the branch `kokkosNewAPI_backend` and this PR will be the base for future development of LightningKokkos with the new API. Another PRs related to this task will be pushed to this branch and once all the work is done then we will do the PR to `master`